article/app/views/liveblog/liveBlogBlock.scala.html

```diff
@@ -31,11 +31,16 @@
         </span>
     </span>
 
-    <p class="block-time published-time">
-        <a href="/@article.metadata.id?page=with:block-@block.id#block-@block.id" itemprop="url" class="block-time__link">
-        @views.html.liveblog.dateBlock(block.referenceDateForDisplay().map(LiveBlogDate(_, timezone)), !isPinnedBlock, true)
-        </a>
-    </p>
+    <div class="block-time">
+        <p class="published-time">
+            <a href="/@article.metadata.id?page=with:block-@block.id#block-@block.id" itemprop="url" class="block-time__link">
+            @views.html.liveblog.dateBlock(block.referenceDateForDisplay().map(LiveBlogDate(_, timezone)), !isPinnedBlock, true)
+            </a>
+        </p>
+        @if(isOriginalPinnedBlock){
+            <span>@fragments.inlineSvg("pin", "icon", List("inline-icon", "pinned-icon-situ", "pinned-block__icon"))</span>
```

Comment:
```suggestion
            <span>@fragments.inlineSvg("pin", "icon", List("inline-icon", "pinned-icon-original", "pinned-block__icon"))</span>
```

Wonder if we should update the word situ here too to reflect the earlier change?

Reply:
Thanks for spotting this Anna, just changed it :)